src/components/Ellipse/Ellipse.stories.tsx

```diff
@@ -5,12 +5,21 @@ import { Story } from "@storybook/react";
 import Ellipse, {EllipseProps} from "./Ellipse";
 
 export default {
-    title:"Icons/Ellipse",
+    title:"Components/Ellipse",
     component:Ellipse
  } as Meta
 
 const Template: Story<EllipseProps>=(args)=><Ellipse {...args} />
 export const Default = Template.bind({})
 Default.args={
     color:"hsla(278, 30%, 48%, 1)"
+    
+}
+export const WithAllProps = Template.bind({})
+
+WithAllProps.args={
+    color:"hsla(278, 30%, 48%, 1)",
+    halo:"hsla(278, 30%, 48%, 1)",
```

Comment:
What is the `halo` doing?

Reply:
Halo is a thing in some special times seen around the moon. It's like a crone or aura. I just checked to google translate for that special word and it gave me that word. Bytheway I changed it to aura:

I have seen it here:

![www figma com_file_msuYC6sv2cOCqZeDtBxNy7_Kerberos io-Apps_node-id=1052%3A45806](https://user-images.githubusercontent.com/36863863/125817540-c5336f24-4752-490d-b63e-2f452d497972.png)



* with aura:

![localhost_9001__path=_story_components-ellipse--with-success-status](https://user-images.githubusercontent.com/36863863/125817103-7b5ad2dc-64fc-4ab4-9f68-934542522336.png)

*without aura:
![localhost_9001__path=_story_components-ellipse--with-success-status (1)](https://user-images.githubusercontent.com/36863863/125817173-7adc6155-4906-4d2c-89e9-b3d5ffc055e6.png)